scripts/report_promotion_closing_keywords.py

```
@@ -0,0 +1,157 @@
+        raise guards.ValidationError(
+            f"git log -1 --format=%B {commit} failed: {result.stdout}{result.stderr}"
+        )
+    return result.stdout
+
+
+def _issue_state(number):
+    """Issue番号の状態を調べる。確認できなければNoneを返す（「CLOSEDである」と決め付けない）。"""
+    if shutil.which("gh") is None:
+        return None
+    result = subprocess.run(
+        ["gh", "issue", "view", number, "--json", "state", "-q", ".state"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)